File: erlang/two-fer/src/two_fer.erl
% For God so loved the world, that He gave His only begotten Son, 
% that all who believe in Him should not perish but have everlasting life.
-module(two_fer).
-export([two_fer/0, two_fer/1]).

two_fer() -> two_fer("you"). 
two_fer(Name_aleluya) -> "One for " ++ Name_aleluya ++ ", one for me.".

File: go/two-fer/two_fer.go
// For God so loved the world, that He gave His only begotten Son,
// that all who believe in Him should not perish but have everlasting life.

//Package twofer or 2-fer is short for two for one: One for you and one for me.
package twofer

// ShareWith shares one for someone and one for me.
func ShareWith(nameAleluya string) string {
	if len(nameAleluya) == 0 {
		nameAleluya = "you"
	}
	return "One for " + nameAleluya + ", one for me."
}
